docs/typescript/virtuals.md

```diff
@@ -82,4 +82,20 @@ const schema = new Schema<UserDoc, UserModel, UserVirtuals>({ // <-- and here
 schema.virtual('fullName').get(function() {
   return `${this.firstName} ${this.lastName}`;
 });
+```
+
+### Override the Type of `this` in Your Virtual
+
+In case the value of `this` in your virtual is incorrect for some reason, you can always override it using the generic parameter to `virtual()`.
```

Comment:
```suggestion
In case the value of `this` in your virtual is incorrect for some reason, you can always override it using the generic parameter in the `virtual()` function.
```

change because i think `to virtual()` does not sound correct